testsuite/integration-tests/pom.xml

```diff
@@ -74,6 +74,13 @@
                                        dest="${project.build.directory}/test-server/wildfly-${server.version}/modules/system/layers/base"
                                        overwrite="true"/>
                                 <delete dir="target/dependency-maven-plugin-markers"/>
+                                <replace file="${project.build.directory}/test-server/wildfly-${server.version}/standalone/configuration/standalone-full.xml">
+                                    <replacetoken><![CDATA[="INFO"]]></replacetoken>
+                                    <replacevalue><![CDATA[="DEBUG"]]></replacevalue>
```

Comment:
We don't need a separate container for tracing test, we just need to switch the log output to DEBUG for tracing test. @asoldano If you agree, I'll keep the solution like this.

Reply:
@liweinan does that mean that the whole testsuite will run with the server having debug level logging enabled? If that's the case, I'd rather avoid this and have a separate container instance for the tracing tests.

Reply:
@asoldano okay then let me create a separate container for this.